CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix broken fragments on CodeSandbox @lucivpav ([#1655](https://github.com/stardust-ui/react/pull/1655))
 - Improve a table with component props in docs @layershifter ([#1634](https://github.com/stardust-ui/react/pull/1634))
 - Improve rendering performance for docs examples @Bugaa92 ([#1654](https://github.com/stardust-ui/react/pull/1654))
+- Fix code in component variables section of theming page @lucivpav ([#1625](https://github.com/stardust-ui/react/pull/1625))
```

Comment:
nit: lets provide an exact name of the docs section in the changelog - otherwise it is not easy to comprehend which page the entry is about:

> Fix code examples in `Theming` page @lucivpav